test_phage_reference/3_validate_expression_data.ipynb

```diff
@@ -516,37 +906,137 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "EColi_pa14_ref_series = expression_pa14_df.loc[\"SRR13234437\"]\n",
+    "nonzero_genes_EColi_pa14_ref = EColi_pa14_ref_series.iloc[EColi_pa14_ref_series.nonzero()]\n",
+    "sns.distplot(nonzero_genes_EColi_pa14_ref, kde=False)\n",
+    "\n",
+    "# Get gene with the max expression\n",
+    "EColi_pa14_ref_series[EColi_pa14_ref_series > 290000]"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Positive case"
+    "**Takeaway:**\n",
+    "\n",
+    "* Most genes were found to have 0 expression, as expected\n",
+    "* Looks like the outlier gene encodes the elongation Tu which appears to be conserved across prokaryotes: https://pubmed.ncbi.nlm.nih.gov/6985898/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Positive case: PAO1\n",
+    "\n",
+    "* Check that PAO1 sample (SRR13160334) aligns well to PAO1 and phage reference. \n",
+    "* Check that PAO1 sample has expression at LUZ19 (NC_010326.1 in phage genome)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
+   "execution_count": 10,
    "metadata": {
-    "scrolled": true
+    "scrolled": false
    },
    "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total number of genes: 5685\n",
+      "Percent of 0 expressed genes: 0.8520668425681618\n",
+      "206449.578392\n"
```

Comment:
I'm not sure what is generating this number!

Reply:
This was because I deleted a line and forgot to rerun the cell